.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,65 @@
+### Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+### Type of Change
+
+- [ ] bugfix
+- [ ] feature
+- [ ] enhancement
+- [ ] refactor
+- [ ] style
+- [ ] docs
+- [ ] chore
+
+### Scope
+
+- [ ] frontend
+- [ ] backend
+- [ ] fullstack
+- [ ] infra
+- [ ] tests
+- [ ] UI/UX
+
+### Priority
+
+- [ ] high-priority
+- [ ] medium-priority
+- [ ] low-priority
```

Comment:
I think for these we may be better off using labels to get more visibility in the pr listing like this

<img width="795" alt="image" src="https://github.com/user-attachments/assets/9b6dac5d-ac8c-4741-8af9-6d5d45494395">